node_tree.py

```diff
@@ -508,6 +508,15 @@ def get_bpy_data_from_name(self, identifier, bpy_data_kind):  # todo, method whi
 
         return None
 
+    def safe_socket_remove(self, kind, key, failure_message=None):
+        with self.sv_throttle_tree_update():
+            sockets = getattr(self, kind)
+            if key in sockets:
+                sockets.remove(sockets[key])
+            else:
+                canned_msg = f"{self.name}.{kind} has no socket named {key} - did not remove"
+                print(failure_message or canned_msg)
```

Comment:
Definitely not print here. And I would appreciate a few string of documentation to such basic method.

Reply:
thanks for your concern, again durman, but i will do this as i like. I will change to self.debug or something if I decide to push. relax man.

Reply:
The thing is that leaving prints everywhere in code is a common mistake. And I don't see why it can't be coded correctly from the beginning.